<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regress</class>
 <widget class="QMainWindow" name="Regress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regress</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Plot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="secondOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Second Order</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">polyGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="thirdOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Third Order</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">polyGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="fourthOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fourth Order</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fifthOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fifth Order</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="firstOrder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>First Order</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Polynomial Fit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="inputTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addRow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeRow">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Row</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="polyGroup"/>
 </buttongroups>
</ui>
